File: Modul-1/C/PRAK107-2310817320007-ZahraNabila.c
#include <stdio.h>

int main()
{
    int sisiTegak, sisiDatar, sisiMiring;

printf("Diketahui : \n");

printf("Masukkan nilai sisi tegak = " );
scanf("%d", &sisiTegak);

printf("Masukkan nilai sisi datar = " );
scanf("%d", &sisiDatar);

printf("Masukkan nilai sisi miring = " );
scanf("%d", &sisiMiring);

    int HargaTanahPerMeter = 85000;
    int KelilingTanah = sisiTegak + sisiDatar + sisiMiring;
    int JumlahPembayaran = HargaTanahPerMeter * KelilingTanah;

printf("Panjang sisi segitiga berturut-turut adalah = %d, %d, %d \n", sisiTegak, sisiDatar, sisiMiring);
printf("Keliling Tanah Pak Dengklek adalah = %d\n", KelilingTanah);
printf("Harga tanah Per Meter = %d\n", HargaTanahPerMeter);

printf("Jawaban : \n");

printf("Biaya yang diperlukan Pak Dengklek adalah : Rp %d\n", JumlahPembayaran);

    return 0;
}

File: Modul-1/Python/PRAK107-2310817320007-ZahraNabila.py
print(f"Diketahui : ")

sisiTegak = int (input("Masukkan nilai sisi tegak = " ))

sisiDatar = int (input("Masukkan nilai sisi datar = " ))

sisiMiring = int(input("Masukkan nilai sisi miring = " ))

HargaTanahPerMeter = 85000
KelilingTanah = sisiTegak + sisiDatar + sisiMiring
JumlahPembayaran = HargaTanahPerMeter * KelilingTanah

print(f"Panjang sisi segitiga berturut-turut adalah = {sisiTegak, sisiDatar, sisiMiring}")
print(f"Keliling Tanah Pak Dengklek adalah = {KelilingTanah}")
print(f"Harga tanah Per Meter = {HargaTanahPerMeter}")

print(f"Jawaban : ")

print(f"Biaya yang diperlukan Pak Dengklek adalah : Rp {JumlahPembayaran}")
